### ISSUE TITLE
What dupe_factor you used?

### ISSUE THREAD

**Issue by ghaddarAbs** [UBC-NLP/marbert#7]
Thank you for sharing the results and model of your great work.

I would like to ask about the duplication factor you used to generate the pretraining data?
(How much you duplicate a sample with different masks).

is it 10 like Google-BERT and AraBERT? or do you set it to a low value like 3 because you have a larger pre-train corpus?

Thanks

**Comment by elmadany** [comment]
Thanks, @ghaddarAbs for your question, we use Google-BERT parameters
